modules/server/src/network/errors.ts

```diff
@@ -0,0 +1,6 @@
+export class NetworkAggregationError extends Error {
```

Comment:
tried and failed to find a way to sugar coat this one, so I'll straight up ask:
this is the first `class` in the whole server codebase. what's the motivation for introducing this "new" pattern? seems like it's meant to satisfy a library requirement e.g. Axios?

Reply:
https://github.com/overture-stack/arranger/pull/926/commits/527e77769ed02d1e16fbd0dbd9fd076f8bf9bdf6
removed